### PR TITLE
fix:edit modal unfiy size

### LINKED ISSUE #1415: Edit subscription dialog changes size with every errors displayed
I would be nice if the edit dialog for subscriptions does not change it size with every single error message, e.g.

Bildschirmfoto 2022-02-19 um 12 39 20
Bildschirmfoto 2022-02-19 um 12 39 47
Bildschirmfoto 2022-02-19 um 12 39 30

### PR DESCRIPTION
**Pull request by sunilsabatp**
Fixes #1415

Changes in this pull request:
- unified the edit donation modal size

### CODE REVIEW

**Comment by norbertschuler** [comment]
- [ ] It's better than before (see screen shots in #1415), but the following dialog is now quite small not fitting all the text anymore:

Bildschirmfoto 2022-03-12 um 14 51 09

**Comment by norbertschuler** [comment]
- [ ] It's better than before (see screen shots in #1415), but the following dialog is now quite small not fitting all the text anymore:

Bildschirmfoto 2022-03-12 um 14 51 09

File: src/features/user/Account/EditModal.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const EditModal = ({
       }}
     >
       <Fade in={editModalOpen}>
-        <div className={styles.manageDonationModal}>
+        <div className={styles.manageDonationModal}  style={{width:"35vw",height:"auto" ,maxWidth:'min-content',overflow:'auto'}} >
```
Comment (norbertschuler) [inline src/features/user/Account/EditModal.tsx:118]: Can we have some more pixels?

**Review by norbertschuler — DISMISSED** [review]

**Comment by tejassonar** [comment]
On mobile view, it looks like this.
We need to use media queries to solve this issue.
image

File: src/features/user/Account/EditModal.tsx (hunk 1 above)
Comment (tejassonar) [reply src/features/user/Account/EditModal.tsx]: Done

**Review by tejassonar — APPROVED** [review]

**Pull request merged**